admin/templates/menu-options.php

```diff
@@ -597,6 +597,20 @@ class="rmp-tab-content__input-checkbox js-rmp-option"
           <?php echo ( esc_html__( 'If enabled all plugin data (settings, customization, vote count, rating and feedback) will be deleted when you uninstall the plugin', 'rate-my-post' ) ); ?>.
         </p>
       </td>
+      <td>
+        <input
+          id="rmp-disable-clear-cache"
+          type="checkbox"
+          data-key="disableClearCache"
+          class="rmp-tab-content__input-checkbox js-rmp-option"
+          <?php echo ($rmp_options['disableClearCache'] === 2) ? 'checked':""; ?>>
+        <label class="rmp-tab-content__label" for="rmp-disable-clear-cache">
+          <?php echo ( esc_html__( 'Disable clear cache', 'rate-my-post' ) ); ?>
+        </label>
+        <p class="rmp-tab-content__notice">
+          <?php echo ( esc_html__( 'If AJAX load results is enabled, this option can be disabled. If you are not utilizing AJAX load results, this option is not recommended. Enable this option if you don\'t want the post cache to be cleared when vote is assigned', 'rate-my-post' ) ); ?>.
```

Comment:
Cambialo così:

If you are not using AJAX load results, this option is not recommended. Enable this option if you don\'t want clear the post cache on every vote submission', 'rate-my-post

Reply:
Fatto